services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/HankeServiceImpl.kt

```diff
@@ -537,6 +555,7 @@ open class HankeServiceImpl(
         hanke.loppuPvm?.let { entity.loppuPvm = hanke.loppuPvm?.toLocalDate() }
 
         hanke.perustaja?.let { entity.perustaja = it.toEntity() }
+        hanke.generated?.let { entity.generated = it }
```

Comment:
Doesn't this allow the user to set the generated value when calling the update endpoint for a hanke?

Reply:
I think there should be a test that tries to set the generated value through the update API.

Reply:
Yes that is a problem. I will find some solution for this but in the long run we should have specific dtos at the controller layer to control what fields the endpoint consumer can affect / see. 

For example the id field used in request bodies to create stuff should probably not even be there.

Reply:
I added functionality to sanitize hanke create and updated requests. Generated is should always be false when creating or updating the hanke.